### ISSUE TITLE
Hover indicator missing for create anything

### ISSUE THREAD

**Issue by barmac** [camunda/camunda-modeler#3435]
### Describe the bug

There is no hover effect for `...` tool. It is present for other tools though.

https://user-images.githubusercontent.com/28307541/217759113-8aadde22-e477-4d55-b998-1167e7193bbe.mov

### Steps to reproduce

1. do this
2. do that
3. now this happens

### Expected behavior

Hover effect is present for each of items.

### Environment

- OS: [e.g. MacOS 10.2, Windows 10]
- Camunda Modeler Version: 5.8.0-rc.0
- Execution Platform: [e.g. Camunda Platform, Camunda Cloud]
- Installed plug-ins: [...]

**Comment by nikku** [comment]
We'd need to show elements as `html` / SVG as it looks like. Then `fill=currentColor` will kick in.

**Comment by barmac** [comment]
Note that the SVG itself seems to not use currentColor.

**Comment by nikku** [comment]
Actually still broken; needs https://github.com/bpmn-io/bpmn-js-create-append-anything/pull/10 to work.

**Comment by nikku** [comment]
Fixed upstream via https://github.com/bpmn-io/bpmn-js-create-append-anything/pull/10 and https://github.com/bpmn-io/bpmn-js-color-picker/pull/27, along with migrating our internal icon loading from image URL to HTML embedding.
